tests/preview_file/custom_file_preview.fish

```diff
@@ -0,0 +1,6 @@
+set multi_word_dir "tests/_resources/multi word dir"
+
+set fzf_preview_file_cmd cat
+
+set actual (__fzf_preview_file "$multi_word_dir/file 1.txt")
+@test "correctly uses custom command to preview files" "$actual" = "This is file 1"
```

Comment:
Hmm this is very good, but the default command, bat, does the same thing as cat...so this test wouldn't fail even if it didn't switch over to cat.

Trying to think of ideas...but I'm tired, I'll come up with something by tomorrow afternoon if you don't by then.

Reply:
This test actually fails with the default, as `bat` is called with `--color=always` among other options. Now I admit that is a bit sideways, but it does in fact work.

Reply:
Maybe something that will print the line in reverse? I thought of tac but it doesn't work b/c it reverses the order of lines, rather than chars with in a line.

Reply:
You could cheat with `echo`ing the path or use `sed` or something to reverse the line, or just add multiple lines and use `tac`. I think I'll leave that up to your preference.

Reply:
Oh, there's `rev`. It might not be available everywhere but if it's available on at least Mac and Ubuntu, then that's good enough for CI, so good enough for me.

Reply:
I didn't see your original comment. Your echo path idea is cogent. Go for it. We're super close to merging.

Reply:
It's in BSD as well as util-linux, so sounds good to me.